doc/modules/feature_extraction.rst

```diff
@@ -321,6 +321,19 @@ requested explicitly::
   ...     ['this', 'is', 'text', 'document', 'to', 'analyze'])
   True
 
+You can customize the way sentences are tokenzied by modifying the
+``token_pattern`` keyword argument with a customized regular expression.
+For example, if you want to include words with apostrophes, you can try:
```

Comment:
I think it's more clear to say "if you do not want to split on apostrophe, you can..."